.travis.yml

```diff
@@ -55,16 +56,16 @@ matrix:
       addons:
           apt:
               sources: *all_sources
-              packages: ['clang-5.0', 'libstdc++-7-dev']
```

Comment:
So we dont support Clang 5 anymore?

Reply:
I feel like if we just keep adding more instances, we'll have to wait forever for each build. So I suggest to remove it at least for now until we figure out how to speed up CI, e.g., until we get a server or something. What do you think?